src/predicates.jl

```diff
@@ -92,40 +92,36 @@ end
 #Base.:(<=)(x::Arf, y::Arf) = !isnan(x) && !isnan(y) && cmp(x, y) <= 0
 #Base.isequal(x::Arf, y::Arf) = !iszero(is_equal(x, y))
```

Comment:
Can you remove these and the other commented out part above now that you are at it?

Reply:
Sure, done.